src/reducers/searchParsers.js

```diff
@@ -247,6 +249,7 @@ function parseAlleleResult(_d) {
     genes: d.genes,
     variantTypes: d.variantTypes,
     molecularConsequence: d.molecularConsequence,
+    relatedVariants: d.relatedVariants,
```

Comment:
Could object spreading be used here to avoid manually enumerating all these properties?